server.js

```diff
@@ -1,94 +1,61 @@
 // Get our dependencies
 var express = require('express');
 var app = express();
-//var mysql = require("mysql");
-//var connection = mysql.createConnection({
-//  host     : process.env.DB_HOST || 'mysql-test.cxrpknmq0hfi.us-west-2.rds.amazonaws.com',
-//  user     : process.env.DB_USER || 'applicationuser',
-//  password : process.env.DB_PASS || 'applicationuser',
-//  database : process.env.DB_NAME || 'movie_db'
-//});
+var mariadb = require("mariadb/callback");
+var connection = mariadb.createConnection({
+  host: process.env.DB_HOST || 'ansrds.cyyo1covxjss.us-east-2.rds.amazonaws.com' || 'localhost',
```

Comment:
please remove the url from the code, search another strategy to replase the host

Reply:
changed added